howtos/release-procedure.md

```diff
@@ -21,20 +21,51 @@ The build process for [publish-prod](https://github.com/camunda/camunda-platform
 
 ## Perform a minor release
 
-Minor releases to Camunda Platform 8 happen twice a year in April and October.
+Minor releases to Camunda Platform 8 happen twice a year in April and October, and the documentation is versioned on the same cadence.
 
-To prepare for a minor release, you'll need to create a new verison.
+To prepare for a minor release, you'll need to create a new version.
```

Comment:
I read this more times than I care to admit before I realized what was going on 🤪

Reply:
Good feedback, means I need to reword it.

Reply:
The typo I mean. This is good to go.

Reply:
oh, well, I was kind of confused by the sentence itself so I added more to it 😄

Reply:
(See https://github.com/camunda/camunda-platform-docs/pull/1334/commits/e0147c7fd438ecbd083bf94dcbd096c9305d1ee4)